CommandLine/emake/EnigmaPlugin.cpp

```diff
@@ -45,7 +45,7 @@ int EnigmaPlugin::Load()
 #endif
 
   std::string pluginName = "./" + prefix + "compileEGMf" + extension;
-  
+
```

Comment:
Try to avoid noising up the blame layer with commits like these...

Reply:
Well, it's mainly fundies fault, his editor (I forget) is worse and the opposite of mine (VS Code). Every time he edits a file his editor just inserts whitespace for the fuck of it.

Though, in this particular instance, blame tells me this one was him, but he did it in one of the initial commits (#1088).

Reply:
I'm going to ask you to let this one slide for two reasons.

First, I am not in favor of adding new files to ENIGMA with formatting mistakes, so I see it as better to fix these formatting mistakes sooner rather than later. Because we're not too many commits into this file's history, and you've raised a stink about it, I am not concerned about the blame being polluted.

Second, I have a new counter-argument against you and in support of this specific kind of formatting change. Nobody ever checks the blame on empty lines, because there's nothing to look for there, so removing excess whitespace is harmless. I get that indentations are another story because those do pollute the blame, but I've not done that here. Also I've now made additional fixes in 3b26f967444d75d01a73c213c9322786f7e3998a to address warnings in the file, so now it's no longer only a formatting change.

So it is for those reasons I am going to be a rebel and go against you this one time.